internal/service/advancedclustertpf/model_flex.go

```diff
@@ -13,6 +13,8 @@ import (
 	"go.mongodb.org/atlas-sdk/v20241113004/admin"
 )
 
+const defaultPriority int = 7
```

Comment:
```suggestion
const defaultPriority = 7
```

no need to say `int`

Reply:
Yes exactly, I needed to put the type to avoid issues when using conversion.Pointer()